src/module.c

```diff
@@ -158,6 +158,7 @@ struct RedisModuleCtx {
     getKeysResult *keys_result;
 
     struct RedisModulePoolAllocBlock *pa_head;
+    long long next_heartbeat_event;
```

Comment:
I might be missing something, but this only ever seems to be set, but never used.

Reply:
it is used in RM_Yield in order to decide when is the next time to call processEventsWhileBlocked